src/ledgerwriter/src/main/resources/application.properties

```diff
@@ -5,3 +5,4 @@ java.util.logging.manager=org.apache.logging.log4j.jul.LogManager
 spring.sleuth.sampler.probability=1.0
 spring.sleuth.web.skipPattern=(^cleanup.*|.+favicon.*)
 spring.sleuth.log.slf4j.enabled=false
+server.tomcat.mbeanregistry.enabled=true
```

Comment:
Why was this needed?

Reply:
This allows all tomcat metrics to be recorded